database/models/events.db.php

```diff
@@ -94,6 +94,9 @@ class Event
     #[Column]
     public string $bulletin_url = "";
 
+    #[Column]
+    public bool $is_simple = false;
```

Comment:
Are you sure we want to just add a boolean here? I was thinking about an event type instead
```suggestion
    public EventType $type = EventType::Multiple;
```
Or something like that.

If we want to extend this bool to cover more types in the future, we will have to do a data migration.

Reply:
Nice as discussed orally we'll remove a bunch of things later